### ISSUE TITLE
How to determine the optimal model?

### ISSUE THREAD

**Issue by gwuei** [wangjiyuan9/WeatherDepth#16]
Hello, I saw that a model is saved at each round. When evaluating each model, I found that different metrics may achieve the best performance on different models. For example, for the abs rel metric, the model from the 10th round might be the best, but for rmse, the model from the 15th round might be the best. But how do we determine the optimal model?

**Comment by wangjiyuan9** [comment]
Epoch is also a hyperparameter, so the standard method is: use the validation set to determine the best model (which is best, depending on yourself and 平衡 each metric), and test with this one.

**Comment by gwuei** [comment]
Understood, thank you for your reply.

**Comment by gwuei** [comment]
May I ask, how do you determine it during training?
